### PR TITLE
TRX: Add support for TRX + USDT.TRX

### PR DESCRIPTION
**Pull request by pablojorge**

### CODE REVIEW

File: chain/tron/tokens.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+[
+    {
+        "address": "1001090",
```
Comment (jjambazov-bc) [inline chain/tron/tokens.json:3]: We do not support TRC10 tokens in HWS. Is this used by non-custodial products?

File: chain/tron/tokens.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        "symbol": "USDD.TRX",
+        "website": "https://usdd.io/"
+    },
+    {
+        "address": "TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t",
+        "decimals": 6,
+        "displaySymbol": "USDT",
+        "logo": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/tron/assets/TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t/logo.png",
+        "name": "Tether",
+        "symbol": "USDT.TRX",
```
Comment (jjambazov-bc) [inline chain/tron/tokens.json:107]: HWS supports only TRX from the whole list of tokens. Are the others supported by non-custodial products?

File: chain/tron/tokens.json (hunk 1 above)
Comment (pablojorge) [reply chain/tron/tokens.json:3]: Yes, this is shared between the assets service (what the wallets consume) and T2

File: chain/tron/tokens.json (hunk 2 above)
Comment (pablojorge) [reply chain/tron/tokens.json:107]: Not yet, but they potentially could, and this repo is supposed to be a repo of all available tokens, not just the ones that the HWS supports

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BCH, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.003 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Review by jjambazov-bc — APPROVED** [review]

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BCH, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [DGLD, COIN]: minWithdrawal 5460 -> $0.010 not in the $0.01-$10 USD range
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.003 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

File: chain/tron/tokens.json [hunk 3]
```diff
@@ -0,0 +1,128 @@
+[
```
Comment (mbratfisch-bc) [inline chain/tron/tokens.json:1]: Where do we took this from? Did you manually build? How the tokens were selected?

**Review by mbratfisch-bc — APPROVED** [review]

File: chain/tron/tokens.json (hunk 3 above)
Comment (pablojorge) [reply chain/tron/tokens.json:1]: We take all assets from https://github.com/trustwallet/assets/tree/a3823f660dd68e0fe99944e931142360cf06e360/blockchains/tron/assets 
We don't select them, we may only disable some if they are in conflict between each other for example

**Pull request merged**